Classes/URKArchive.h

```diff
@@ -234,6 +234,22 @@ extern NSString *URKErrorDomain;
 + (BOOL)urlIsARAR:(NSURL *)fileURL;
 
 /**
+ *  Determines whether the current file is a RAR volume part
```

Comment:
Just to be clear, can't use `isVolume` because it does not detect as a volume the first part. I'm removing this method.

https://github.com/abbeycode/UnrarKit/pull/38#issuecomment-277518409
> ...Sadly I did not found any other way to get the first part of the volume detected as a volume, nor get a consistent way to detect the next/previous volume existence. So right now the code is as here.